content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,220 @@
+
+### Configuring Agent integrations
+
+Configurations are specific to [individual integrations][18]. In the `conf.d` folder at the root of your Agent's configuration directory, there is a file named `<INTEGRATIONS>.d` in each integration folder, and a sample `conf.yaml.example` that lists all available configuration options. 
+
+To begin configuring, rename `conf.yaml.example` (in `<INTEGRATIONS>.d`) to `conf.yaml` to activate the integration. Then update the parameters inside the configuration file. Non-required parameters are commented out. All configuration files follow the format documented in the [parameters documentation][22].
+
+All configuration files have things in common:
+* Not required parameters are commented out of the example files.
+* If the integration supports our Logging solution, you configure it here too.
+* You can set up multiple instances in the same file to monitor local host and remote endpoints.
+
+For example, this is a sample `conf.yaml` for the `apache` integration:
+
+```
+init_config:
+
+instances:
+
+  ## @param apache_status_url - string - required
+  ## Status url of your Apache server.
+  #
+  - apache_status_url: http://localhost/server-status?auto
+
+  ## @param apache_user - string - optional
+  ## Username for the Apache status endpoint authentication.
+  #
+  #  apache_user: <USERNAME>
+
+  ## @param apache_password - string - optional
+  ## Password for the Apache status endpoint authentication.
+  #
+  #  apache_password: <PASSWORD>
+
+  ## @param tags  - list of key:value elements - optional
+  ## List of tags to attach to every metric, event and service check emitted by this integration.
+  ##
+  ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+  #
+  #  tags:
+  #    - <KEY_1>:<VALUE_1>
+  #    - <KEY_2>:<VALUE_2>
+
+  ## @param disable_ssl_validation - boolean - optional - default: false
+  ## Instructs the check to skip the validation of the SSL certificate of the URL being tested.
+  ## Defaults to false, set to true if you want to disable SSL certificate validation.
+  #
+  #  disable_ssl_validation: false
+
+  ## @param connect_timeout - integer - optional
+  ## Overrides the default connection timeout value,
+  ## and fails the check if the time to establish the (TCP) connection
+  ## exceeds the connect_timeout value (in seconds)
+  #
+  #  connect_timeout: <VALUE_IN_SECOND>
+
+  ## @param receive_timeout - integer - optional
+  ## Overrides the default received timeout value, and fails the check if the time to receive
+  ## the server status from the Apache server exceeds the receive_timeout value (in seconds)
+  #
+  #  receive_timeout: <VALUE_IN_SECOND>
+
+## Log Section (Available for Agent >=6.0)
+##
+## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel if type is windows_event
```

Comment:
I understand why this is all on one line, but when we render it on the docs site it's visually problematic:
![image](https://user-images.githubusercontent.com/625232/55862689-40551900-5b79-11e9-8d29-4bed6122190c.png)

I'm not sure what a solution (if any) is since the standard requires single lines for descriptions. 😕 cc @l0k0ms

Reply:
I think we should crop this example, giving only a reference to the source file at the beginning and give the minimum required configuration for metrics + logs while we remove paramters comments. And instead add some comments like 

`required configuration for metrics collection` and `required configuration for logs collection` or somthing

then create an other example just below where we define two apache check instances.